src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,9 +1,46 @@
-import React, { ComponentProps } from 'react';
+import React, { ComponentProps, useState } from 'react';
+import { PopperProps, usePopper } from 'react-popper';
+
+export const Tooltip = ({ children, ...rest }: ComponentProps<'div'>) => (
+  <div className='str-chat__tooltip' {...rest}>
+    {children}
+  </div>
+);
+
+export const PopperTooltip = <T extends HTMLElement>({
+  children,
+  offset = [0, 10],
+  referenceElement,
+  placement = 'top',
+  visible: visible = false,
+}: React.PropsWithChildren<{
```

Comment:
It would be great to document the props purpose with comments. Wouldn't it be more robust to keep them in a separate type?